inc/genrenator.php

```diff
@@ -69,6 +69,8 @@ function generate_genre() {
 	$patterns = patterns();
 	$index    = array_rand( $patterns );
 
+	update_option( 'generate_genre', get_option( 'generate_genre', 0 ) + 1, false );
+
```

Comment:
I feel like if we’re adding an option here, we should have an action hook along with it.